extensions/amp-story-interactive/0.1/amp-story-interactive-img-poll.js

```diff
@@ -20,7 +20,31 @@ import {
 } from './amp-story-interactive-abstract';
 import {CSS} from '../../../build/amp-story-interactive-img-poll-0.1.css';
 import {CSS as ImgCSS} from '../../../build/amp-story-interactive-img-0.1.css';
+import {buildImgTemplate} from './utils';
 import {htmlFor} from '#core/dom/static-template';
+import {setStyle} from '#core/dom/style';
+
+/**
+ * Generates the template for each option.
+ *
+ * @param {!Element} option
+ * @return {!Element}
+ */
+const buildOptionTemplate = (option) => {
+  const html = htmlFor(option);
+  return html`
+    <button
+      class="i-amphtml-story-interactive-img-option i-amphtml-story-interactive-option"
+      aria-live="polite"
+    >
+      <div class="i-amphtml-story-interactive-img-option-img">
+        <span
+          class="i-amphtml-story-interactive-img-option-percentage-text"
+        ></span>
```

Comment:
I would suggest removing this from this PR since styling and logic for it is not included in this PR.